### ISSUE TITLE
Add links for the apps website

### ISSUE THREAD

**Issue by Mazyod** [SwiftKitz/Appz#22]
We should link the apps in the readme file to publisher website, for easy referencing.

**Comment by Maryom** [comment]
@Mazyod I read this comment a lot but I couldn't get it. Could you please give an example? do you mean website for the references like for example: http://tapbots.net/tweetbot3/support/url-schemes/

**Comment by Mazyod** [comment]
@Maryom For this issue, I would like to add links to the README file. For example, in the readme we would have:

```
...
[Facebook][facebook-link] | Open, Profile, Notification, Feed
...

[facebook-link]: https://facebook.com
```

Which will allow developers to see the publisher website.

**Comment by Maryom** [comment]
@Mazyod and for app that doesn't have website e.g. setting we will leave it empty?

**Comment by Mazyod** [comment]
@Maryom No, it's better to have something that explains that as well:

[AppSettings](https://developer.apple.com/library/ios/documentation/UIKit/Reference/UIApplication_Class/index.html#//apple_ref/c/data/UIApplicationOpenSettingsURLString)

**Comment by Maryom** [comment]
@Mazyod Ok deal, I will try to add them ASAP.

**Comment by Mazyod** [comment]
Thanks, you've been assigned!

**Comment by Maryom** [comment]
@Mazyod You are welcome.

**Comment by Maryom** [comment]
@Mazyod Cool I just know that there is an assignee in the right side, Awesome I loved it!

**Comment by Mazyod** [comment]
closed by #24
